### PR TITLE
chore: Added Security MD file to the repo

### PR DESCRIPTION
**Pull request by jebrando**

### CODE REVIEW

File: SECURITY.MD [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.3 BLOCK -->
```
Comment (olivakar) [inline SECURITY.MD]: Do we need to have additional begin and end blocks ?

It does not provide an additional format style.

Comment (jebrando) [reply SECURITY.MD]: I don't know, I was scared to remove since we got it from MSRC

**Pull request merged**